tests/end-to-end/tests/barchart.cy.ts

```diff
@@ -0,0 +1,50 @@
+import { selectors, testIds } from '../support/const';
+const paths = {
+  barChart: 'ComponentOverlay_rootcontainer/maincontainer/pagesection/barchart'
+};
+
+describe('barChart Component', () => {
```

Comment:
need to update all describes to be consistent so that you can tell where the test is, seem so be random patterns atm.

Reply:
OK